tests/components/squeezebox/test_config_flow.py

```diff
@@ -166,6 +166,75 @@ async def patch_async_query(self, *args):
     assert result["errors"] == {"base": "invalid_auth"}
 
 
+async def test_form_validate_exception(hass: HomeAssistant) -> None:
+    """Test we handle exception."""
+
+    with (
+        patch(
+            "pysqueezebox.Server.async_query",
+            return_value={"uuid": UUID},
+        ),
+        patch(
+            "homeassistant.components.squeezebox.async_setup_entry",
+            return_value=True,
+        ),
+        patch(
+            "homeassistant.components.squeezebox.config_flow.async_discover",
+            mock_discover,
+        ),
+    ):
+        result = await hass.config_entries.flow.async_init(
+            DOMAIN, context={"source": config_entries.SOURCE_USER}
+        )
+        assert result["type"] is FlowResultType.FORM
+        assert result["step_id"] == "edit"
+        assert CONF_HOST in result["data_schema"].schema
+        for key in result["data_schema"].schema:
+            if key == CONF_HOST:
+                assert key.description == {"suggested_value": HOST}
```

Comment:
```suggestion
```
We don't have to check this in every test

Reply:
removed